bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -31,18 +34,31 @@ The binding has the following configuration options:
 | readFriends  | Boolean       | Enables or disables the discovery of guest weather stations.                               |
 
 
-## Bridge Configuration
+## Netatmo Account (Bridge) Configuration
 
 You will have to create at first a bridge to handle communication with your Netatmo Application.
 
-The Account bridge has the following configuration options:
+The Account bridge has the following configuration elements:
+
+| Parameter         | Type   | Required | Description                                                                                                            |
+|-------------------|--------|----------|------------------------------------------------------------------------------------------------------------------------|
+| clientId          | String | Yes      | Client ID provided for the application you created on http://dev.netatmo.com/createapp                                 |
+| clientSecret      | String | Yes      | Client Secret provided for the application you created                                                                 |
+| webHookUrl        | String | No       | Protocol, public IP and port to access openHAB server from Internet                                                    |
+| reconnectInterval | Number | No       | The reconnection interval to Netatmo API (in s)                                                                        |
+| refreshToken      | String | No       | The refresh token provided by Netatmo API after the granting process. Can be saved in case of file based configuration |
```

Comment:
If you use file based configuration, this is actually a required field. If you don't configure it. You need to reauthorize when oh is restarted.

Reply:
Yes but I can not set it required in the config-description, or else it will not be possible to create the thing in UI. So it is not mandatory for thing creation but needed for smooth operations :). I will update the doc accordingly.